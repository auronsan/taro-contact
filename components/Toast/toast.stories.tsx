import React from 'react';
import { Toast, ToastContainer } from './index';

export default {
  title: 'Components/Toast',
  component: Toast,
};

export const Default = () => (
  <ToastContainer>
    <Toast message="Default Toast" onDismiss={() => {}} />
  </ToastContainer>
);

export const Success = () => (
  <ToastContainer>
    <Toast message="Success Toast" onDismiss={() => {}} type="success" />
  </ToastContainer>
);

export const Error = () => (
  <ToastContainer>
    <Toast message="Error Toast" onDismiss={() => {}} type="error" />
  </ToastContainer>
);

export const WithDuration = () => (
  <ToastContainer>
    <Toast message="Toast with Duration" onDismiss={() => {}} duration={5000} />
  </ToastContainer>
);
