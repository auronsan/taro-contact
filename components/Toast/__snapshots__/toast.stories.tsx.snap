// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Toast Default smoke-test 1`] = `
<main class="roboto-normal">
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
    <div class="toast_toast__tBw0J toast_success__TUm0C">
      Default Toast
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`Components/Toast Error smoke-test 1`] = `
<main class="roboto-normal">
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
    <div class="toast_toast__tBw0J toast_error__SGlCN">
      Error Toast
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`Components/Toast Success smoke-test 1`] = `
<main class="roboto-normal">
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
    <div class="toast_toast__tBw0J toast_visible___6NKv toast_success__TUm0C">
      Success Toast
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`Components/Toast WithDuration smoke-test 1`] = `
<main class="roboto-normal">
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
    <div class="toast_toast__tBw0J toast_success__TUm0C">
      Toast with Duration
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
